<launch>

<include file="$(find nuslam)/launch/view_diff_drive_slam.launch"/>
<rosparam command="load" file="$(find tsim)/config/waypoint_x.yaml" />  
<rosparam command="load" file="$(find tsim)/config/waypoint_y.yaml" />  
<rosparam command="load" file="$(find nuturtle_robot)/config/nubot_params.yaml"/> 
   
<node  name="odometer1" pkg="rigid2d" type="odometer" output = "screen">
<param name="world" value="odom" />
<param name="base" value="base_link" />
</node>

<node pkg="tf" type="static_transform_publisher" name="odom_link_broadcaster" args="0 0 0 0 0 0  /odom /fake_odom 100" />

<node name="turtle_interface" pkg="nuturtle_robot" type="turtle_interface" />

</launch>
